query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/bytes_filter.rs

```diff
@@ -63,6 +63,11 @@ mod bytes_filter_spec {
           @r###"{"data":{"findManyTestModel":[{"id":1}]}}"###
         );
 
+        insta::assert_snapshot!(
```

Comment:
Please add a negation test for completeness as well.

Reply:
done